docs/walkthroughs/developing-apbs-locally.adoc

```diff
@@ -134,35 +223,16 @@ you can force the broker to reload images from your org.
 apb bootstrap
 ----
 
-== Using the mobile-service tag
-
-For MCP enabled _APBs_, make sure we have a `mobile-service` tag inside the `apb.yml` file:
-
-....
-...
-tags: 
-  - mobile-service
-....
-
-== Using the servicename label
-
-Inside of the `apb.yml` file, make sure you use the `serviceName:` label, like:
-
-....
-...
-metadata:
-  displayName: Aerogear Sync Server
-  console.openshift.io/iconClass: icon-nodejs
-  serviceName: fh-sync-server
-...
-....
-
 == Testing an APB
 
-Currently, there are not many tools for testing. The APB team has a few open Github issues and proposals:
+Good practice is to have a special playbook for testing, called ``test.yml``. This playbook is used for quick verification of the 
+implemented roles within the APB repository. If you include some/all roles (provision, bind, deprovision, unbind) in your
```

Comment:
did you mean `If you include some/all roles ... in your test playbook`?

Reply:
No, i'm saying if you have a provision, bind etc, task as part of your service it should have a matching test for that task.

I could make that clearer.

Reply:
Ah ok, i guess the word `service` as unclear to me because i was thinking of either `apb` or `test`